src/helpers/types.ts

```diff
@@ -398,6 +404,18 @@ export enum OverdraftApplicationDecision {
   REJECTED = "REJECTED",
 }
 
+export enum DeliveryMethod {
+  MOBILE_NUMBER = '"mobile_number"',
+  DEVICE_SIGNING = "device_signing",
```

Comment:
is it `device_signing` or `device_binding`?

Reply:
<img width="351" alt="Screen Shot 2021-04-14 at 11 22 07" src="https://user-images.githubusercontent.com/10659918/114687129-b6b22600-9d13-11eb-9c6e-4ede0929c7b4.png">

Reply:
then it's different from PDF doc